dev-libs/sblim-sfcc/sblim-sfcc-2.2.8.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Small Footprint CIM Client Library"
+HOMEPAGE="https://sourceforge.net/apps/mediawiki/sblim"
+SRC_URI="mirror://sourceforge/project/sblim/${PN}/${P}.tar.bz2"
+
+LICENSE="EPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	net-misc/curl[ssl]
+	"
+RDEPEND="${DEPEND}"
+DOCS=()
+
+src_configure() {
+	econf --enable-http-chunking \
+	--disable-static
```

Comment:
Please either indent following lines to make it clear they're continuation or preferably use `local myconf=()` array to avoid backslash hell.